Amazon.QLDB.Driver.IntegrationTests/AsyncStatementExecutionTests.cs

```diff
@@ -852,20 +494,101 @@ await foreach (IIonValue row in result)
             var timingInfo = result.GetTimingInformation();
 
             Assert.IsNotNull(ioUsage);
             Assert.IsNotNull(timingInfo);
+
             // The 1092 value is from selectQuery, that performs self joins on a table.
             Assert.AreEqual(1092, ioUsage?.ReadIOs);  
             Assert.IsTrue(timingInfo?.ProcessingTimeMilliseconds > 0);
         }
+        
+        private static async Task<int> ExecuteAndReturnRowCount(string statement)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement);
+
+                return await result.CountAsync();
+            });
+        }
+        
+        private static async Task<int> ExecuteWithParamAndReturnRowCount(string statement, IIonValue param)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement, param);
+                
+                return await result.CountAsync();
+            });
+        }
+        
+        private static async Task<int> ExecuteWithParamsAndReturnRowCount(string statement, List<IIonValue> parameters)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement, parameters);
+                
+                return await result.CountAsync();
+            });
+        }
+
+        private static async Task<IIonValue> ExecuteAndReturnIonValue(string statement)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement);
+
+                IIonValue value = null;
+                await foreach (var row in result)
+                {
+                    value = row;
+                }
+                return value;
+            });
+        }
+        
+        private static async Task<IIonValue> ExecuteWithParamAndReturnIonValue(string statement, IIonValue param)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement, param);
 
+                IIonValue value = null;
+                await foreach (var row in result)
+                {
+                    value = row;
+                }
+                return value;
+            });
+        }
+        
+        private static async Task<IIonValue> ExecuteAndReturnField(string statement, string fieldName)
+        {
+            return await qldbDriver.Execute(async txn =>
+            {
+                var result = await txn.Execute(statement);
+
+                IIonValue ionValue = null;
+                await foreach (var row in result)
+                {
+                    ionValue = row.GetField(fieldName);
+                }
+                return ionValue;
+            });
+        }
+        
+        private static async Task<bool> ConfirmTable(string tableName)
```

Comment:
Since this checks the passed table in all of the tables, how about `confirmTableExists()`

Reply:
Done.